app-libs/form-component/src/layout-components/Dropdown/DropdownLayout.module.css

```
@@ -0,0 +1,62 @@
+  display: flex;
+  flex-direction: column;
+}
+
+.visuallyHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)